x-pack/plugins/security_solution/server/lib/telemetry/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { ISavedObjectsRepository } from '../../../../../../src/core/server';
+import { getEndpointTelemetryFromFleet, EndpointUsage } from './endpoint';
+
+export interface UsageData {
+  endpoint?: EndpointUsage;
+}
+
+export async function createSiemTelemetry(
+  usageCollector: UsageCollectionSetup,
+  savedObjectsClient: ISavedObjectsRepository
+) {
+  const collector = usageCollector.makeUsageCollector<UsageData>({
+    type: 'security_solution',
+    isReady: () => true,
+    fetch: async () => {
+      try {
+        return {
+          endpoint: await getEndpointTelemetryFromFleet(savedObjectsClient),
+        };
+      } catch (err) {
+        return {};
+      }
+    },
+    schema: {
+      endpoint: {
+        total_installed: { type: 'number' },
```

Comment:
NIT: To help us out, it might be beneficial if devs specify `'long'` instead.

Looks good as-is anyway. No need to change it 👍

Reply:
Sweet, I can change it though. No biggie. Once @rylnd has his PR in I'll rebase on his and update these to `long` 😄